src/main/config/config.c

```diff
@@ -660,11 +666,6 @@ static void resetConf(void)
 
 #if defined(ALIENFLIGHT) 
     featureClear(FEATURE_ONESHOT125);
-#ifdef ALIENFLIGHTF1
-    masterConfig.serialConfig.portConfigs[1].functionMask = FUNCTION_RX_SERIAL;
-#else
-    masterConfig.serialConfig.portConfigs[2].functionMask = FUNCTION_RX_SERIAL;
```

Comment:
Will get this done likely tomorow when I'm back home. The firmware is getting very nicely structured now.